web/portal/user/src/entities/Preferences/ForeignKeyGetter.tsx

```diff
@@ -0,0 +1,40 @@
+import { ForeignKeyGetterType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { autoSelectOptions } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { UserPropertyList } from '../User/UserProperties';
+import entities from '../index';
+import TimezoneSelectOptions from 'entities/Timezone/SelectOptions';
+
+export const foreignKeyGetter: ForeignKeyGetterType = async ({
+  cancelToken,
+  entityService,
+  match,
+}): Promise<any> => {
+  const response: UserPropertyList<unknown> = {};
+
+  const promises = autoSelectOptions({
+    entities,
+    entityService,
+    cancelToken,
+    response,
+    skip: [],
```

Comment:
`skip: ['timezone']` or remove TimezoneSelectOptions call below
It think you're doing the request twice here.